### PR TITLE
Add test on GitHub Actions

### PR DESCRIPTION
**Pull request by hoh**
Following the [NixOS guide on Continuous Integration with GitHub Actions](https://nixos.org/guides/continuous-integration-github-actions.html), this builds the APK on GitHub Actions.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    #     signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+    #     # If you chose API tokens for write access OR if you have a private cache
+    #     authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+
+    # - run: nix-build
+    - run: nix-shell --run "echo OK"
+      env: 
+        NIXPKGS_ALLOW_UNFREE: 1
+
+    - run: nix-shell --run "flutter build apk --ignore-deprecation"
```
Comment (babariviere) [inline .github/workflows/test.yml:27]:
```suggestion
    - run: nix-shell --run "flutter build apk"
```

Comment (babariviere) [reply .github/workflows/test.yml:27]: We don't need the ignore-deprecation in our case.

Comment (hoh) [reply .github/workflows/test.yml:27]: Indeed, I added it before your upgrade.

**Comment by babariviere** [comment]
Thanks!

**Pull request merged**